core/src/utils/input-masking/utils/format.spec.ts

```diff
@@ -0,0 +1,18 @@
+import { formatMask } from './format';
+
+describe('formatMask', () => {
+  it('should convert a string to a RegExp', () => {
+    const mask = formatMask('0');
+    expect(mask).toBeInstanceOf(RegExp);
```

Comment:
Could we make this check for the exact regex to be safe?

Reply:
Now that our API signature is consistent (`RegExp` instead of `string`), `formatMask` isn't needed so I deleted it and the respective tests 🎉